desktop/apps/auction/components/basic_checkbox/index.js

```diff
@@ -1,13 +1,13 @@
-import classNames from 'classnames'
+import PropTypes from 'prop-types'
 import React from 'react'
+import classNames from 'classnames'
```

Comment:
Ah yeah, most editors come with different sort strategies behind the command panel